backends/ebpf/CMakeLists.txt

```diff
@@ -87,13 +87,13 @@ set (XFAIL_TESTS_BCC)
 set (XFAIL_TESTS_TEST)
 
 set (EBPF_TEST_SUITES
-  "${P4C_SOURCE_DIR}/testdata/p4_16_samples/*_ebpf.p4"
+  "${P4C_SOURCE_DIR}/testdata/p4_16_samples/ebpf/*_ebpf.p4"
```

Comment:
Why are you moving these files around?
This way you are affecting removing these tests from the p4test compiler test path, so you are removing many useful tests.

Reply:
The intention was to clean up the testdata folder a bit and sort the tests into more explicit categories, to make it clearer and easier to navigate in the folder. I was not aware that p4test depended on it, sorry. I can revert it.

Reply:
How about using file instead of set? Using GLOB_RECURSE files in subdirectories are also added.
For example:
file(GLOB_RECURSE P4TEST_SUITES
  "${P4C_SOURCE_DIR}/testdata/p4_16_samples/*.p4"
  "${P4C_SOURCE_DIR}/testdata/p4_16_errors/*.p4"
  )
That way the tests remain in the compiler path and can also be sorted into categories.